src/app/lib/utilities/getToggles/index.js

```diff
@@ -43,7 +53,7 @@ const getToggles = async (service, cache) => {
       ...toggles,
     };
   } catch (error) {
-    logger.error(CONFIG_FETCH_ERROR, { error: error.toString(), url, service });
+    logger.error(CONFIG_ERROR, { error: error.toString(), url, service });
```

Comment:
These changes look great, don't forget to update the runbook so people can look for these errors during a live incident 👍

Reply:
Just writing up an issue to add these to the runbook and to sumologic :+1: